physionet-django/console/views.py

```diff
@@ -1493,3 +1522,32 @@ def get_queryset(self):
             qs = qs.filter(username__icontains=self.q)
 
         return qs
+
+@login_required
+@user_passes_test(is_admin, redirect_field_name='project_home')
+def known_referencess(request):
+    """
+    List all known references witht he option of removing the contact date
+    """
+    user = request.user
+
+    if 'remove_known_ref' in request.POST and \
+     request.POST['remove_known_ref'].isdigit():
```

Comment:
this is formatted oddly (there is a single space on the second line, and i think it is best to avoid `\` to break lines. wrapping the clause with `(` .. `)` would work and i think conforms to pep8.

Reply:
For some reason my pep8 lint is broken... will have to see why...

Reply:
If using the (...) by pep8 that I fixed it I would have the following:
```
    if ('remove_known_ref' in request.POST and
       request.POST['remove_known_ref'].isdigit()):
        try:
```
VS 
```
    if 'remove_known_ref' in request.POST and \
       request.POST['remove_known_ref'].isdigit():
        try:
```
is the same really in terms of spaces, but I think we use more the \ than the (...) so I will add more spaces